config.w32

```diff
@@ -58,44 +58,50 @@ if (PHP_MONGODB != "no") {
   ADD_EXTENSION_DEP("mongodb", "openssl", false);
 
   var PHP_MONGODB_CFLAGS="\
-    /D BSON_COMPILATION /D MONGOC_COMPILATION /D MONGOC_TRACE \
+    /D BSON_COMPILATION /D MONGOC_COMPILATION \
```

Comment:
Did libmongoc remove the tracing?

Reply:
This was moved to a `mongoc-config.h` constant.